Components/modal/modal.css

```diff
@@ -0,0 +1,42 @@
+.modal {
+  display: flex;
+  background-color: var(--snackbar-blackbg);
+  padding: 1rem;
+  width: 40rem;
+  height: 4rem;
+  justify-content: space-around;
+  align-items: center;
+  color: var(--white-text-color);
+}
+
+.modal-bg {
+  position: fixed;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: 100vh;
+  background-color: rgb(0, 0, 0, 0.5);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  visibility: hidden;
+  opacity: 0;
+}
+
+.bg-active {
+  visibility: visible;
+  opacity: 1;
+}
+
+.close-modal {
+  color: var(--white-text-color);
+  font-size: 28px;
+  font-weight: bold;
+  top: 5px;
+}
+
+.close-modal:hover,
+.close-modal:focus {
+  color: black;
```

Comment:
use variables or hex code for colors, as different browser interprets color differently.